### DISCUSSION TITLE
development kit using the KSZ9031RNX PHY

### DISCUSSION THREAD

**Discussion by rockoguido** [polarfire-soc/polarfire-soc-documentation#288] [Reference designs, FPGA and development kits]
Hi..  looking for a eval kit that makes use of the development kit using the KSZ9031RNX PHY.    Our PolarFire Eval kit uses the VSC8575.
the motivation for asking is that I'm trying to get ethernet working on a 3rd party board that uses this particular Microchip PHY.  

Thx!
Dave

**Comment by nitindeshpande** [comment]
@rockoguido Did you find the kit you were looking for? Could you please update the tread either way so it can be concluded?

**Reply by rockoguido** [reply]
have not found such a kit priority has shifted, still would be good to find but the importance has diminished
